content/en/notebooks/advanced_analysis/_index.md

```diff
@@ -0,0 +1,122 @@
+---
+title: Analysis Features
+further_reading:
+- link: "/notebooks"
+  tag: "Documentation"
+  text: "Learn more about Notebooks"
+---
+{{< callout url="#" header="Join the Preview!">}} 
+  Advanced Analysis is in Preview
+{{< /callout >}}
+
+## Overview
+
+The computational functionality within Notebooks supports complex operations, such as combining attributes from multiple sources or transforming data, while still providing all the powerful features that Notebooks offer. 
+
+Notebooks are collaborative text editors that let you embed Datadog graphs directly into your documents. While this is great for exploration and storytelling, investigations may require more advanced control over data queries. Use analysis features to run queries that help you:
+
+* Correlate multiple data sources  
+* Aggregate multiple levels of data  
+* Join data across multiple log sources and other datasets  
+* Extract data or add a calculated field at query time  
+* Visualize transformed datasets with embedded graphs
+
+## Adding data to your notebook
+
+To run complex queries in a notebook, you first need to add a **Data Source** cell. There are two ways to do this: 
+
+**From a notebook**:
+1. Type `/datasource` and press <kbd>Enter</kbd>, or click the **Data Source** tile at the bottom of the page.  
+2. Type or select your desired data source from the drop down menu and press <kbd>Enter</kbd>.  
+3. Enter your query. Reserved attributes from the filtered logs are automatically added as columns.
+
+**From the [Log Explorer][1]**:
+
+1. Enter your query in the Log Explorer.  
+2. Click **Analyze in Notebooks**.  
+3. Check the **Use as a computational data source** box and select the notebook you want to use.   
+4. A data source cell is added to the selected notebook with the same query you entered in the Log Explorer. By default, the columns shown in the Log Explorer are included in the data source cell.
+
+## Configuring a data source cell
+
+After adding a data source cell to a notebook, you can continue modifying it to structure the data to suit your analysis needs. 
+
+### Change the time frame for the data
+
+By default, data source cells created from Notebooks use the global time frame. Data source cells created from the Log Explorer use a local time fixed to the time frame at the time of export.
```

Comment:
Should say "use the notebook's global time frame"

Reply:
@joparisot To confirm, is this what you want changed:
```suggestion
By default, data source cells created from Notebooks use the notebook's global time frame. Data source cells created from the Log Explorer use a local time fixed to the time frame at the time of export.
```

Reply:
yes